<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="41179" started-at="2017-03-27T14:17:41Z" finished-at="2017-03-27T14:18:22Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="41179" started-at="2017-03-27T14:17:41Z" finished-at="2017-03-27T14:18:22Z">
      <class name="main.java.com.dbyl.tests.Test">
        <test-method status="FAIL" signature="myTest()[pri:0, instance:main.java.com.dbyl.tests.Test@52e6fdee]" name="myTest" duration-ms="40615" started-at="2017-03-27T22:17:41Z" finished-at="2017-03-27T22:18:22Z">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[while trying to invoke the method main.java.com.dbyl.libarary.utils.Locator.getElement() of a null object loaded from local variable 'locator']]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: while trying to invoke the method main.java.com.dbyl.libarary.utils.Locator.getElement() of a null object loaded from local variable 'locator'
	at main.java.com.dbyl.libarary.utils.BasePage.findElement(BasePage.java:281)
	at main.java.com.dbyl.libarary.utils.BasePage$1.apply(BasePage.java:245)
	at main.java.com.dbyl.libarary.utils.BasePage$1.apply(BasePage.java:1)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:209)
	at main.java.com.dbyl.libarary.utils.BasePage.isElementPresent(BasePage.java:241)
	at main.java.com.dbyl.libarary.utils.BasePage.isElementPresent(BasePage.java:267)
	at main.java.com.dbyl.tests.Test.myTest(Test.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- myTest -->
      </class> <!-- main.java.com.dbyl.tests.Test -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
